secretmanager/api-client/access_secret_version.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google, Inc
```

Comment:
```suggestion
# Copyright 2019 Google LLC
```

(same elsewhere)

Reply:
Hmm - both storage and kms (which is where I copied the license headers from) use Inc. Is this a recent change to use LLC? Should the canonical example (storage) be updated?

Reply:
IIRC the change occurred in 2018, but it's only applied to new files. 

We do have some new canonical and updates to the contributing guide in the pipeline.